tools/xdmod-ondemand-export/xdmod_ondemand_export/xdmod_ondemand_export.py

```diff
@@ -24,30 +24,29 @@ def __init__(self):
         # Override how exceptions are handled.
         sys.excepthook = self.__excepthook
         self.__logger.debug('Using arguments: ' + str(self.__args))
-        self.__validate_file_permissions()
+        self.__validate_file_permissions_600(
+            os.path.expanduser('~/xdmod-ondemand-export/.token')
```

Comment:
Not sure its a good idead to hard code the `~/xdmod-ondemand-export` path. The README doesn't make it clear that you _have_ to use ~/xdmod-ondemand-export as the package dir and it won't work if you choose something different.

You can use python to tell you where the package resources have been installed - thus not hardcoding it:

See example code at:

https://github.com/ubccr/supremm/blob/master/src/supremm/config.py#L62

Reply:
Another alternative is to have a 'default' value in the software (like you have) but also add a command line argument to override the default.

Reply:
Also I don't think we want to try to enforce 600 permissions for the token file. It only needs read-only for the user account that runs the process.

Reply:
I have added a command line argument to override the token path.

The script prints warnings if the file permissions are not 600 on the token and/or JSON file, but it goes ahead and runs anyway. Should we remove those warnings?